src/Aurora.Infrastructure/Scrapers/ScraperBase.cs

```diff
@@ -63,6 +81,24 @@ public async Task<ExtendedSearchResult> PerformSearch(SearchRequest request, Can
             };
         }
 
-        public abstract Task<ValueOrNull<SearchResult>> SearchInner(SearchRequest request, CancellationToken token = default);
+        public virtual Task<ValueOrNull<SearchResult>> SearchVideosInner(SearchRequest request, CancellationToken token = default)
+        {
+            return default;
+        }
+
+        public virtual Task<ValueOrNull<SearchResult>> SearchImagesInner(SearchRequest request, CancellationToken token = default)
+        {
+            return default;
```

Comment:
default of ValueOrNull returns object where value is null, isNull == false. Please, modify this class or return null, which would get properly converted to valueOrNull (it cannot be null itself, it is a struct)